dev-libs/tinyxml2/files/tinyxml2-9.0.0-musl-1.2.4.patch

```diff
@@ -0,0 +1,64 @@
+Bug: https://bugs.gentoo.org/905999
+Upstream: https://leethomason/tinyxml2/pull/945
+
+From dfcb914e8bbbb2dca146a3cee62f66fff7ea163b Mon Sep 17 00:00:00 2001
+From: Violet Purcell <vimproved@inventati.org>
+Date: Thu, 8 Jun 2023 15:19:41 +0000
+Subject: [PATCH] Remove LFS64 calls and set _FILE_OFFSET_BITS=64
+
+Musl 1.2.4 made the LFS64 interfaces only available when
+_LARGEFILE64_SOURCE is defined, and they will be removed altogether in
+Musl 1.2.5. This commit replaces the LFS64 calls with their non-LFS64
+versions and defines _FILE_OFFSET_BITS=64, which makes all interfaces
+64-bit.
+
+Bug: https://bugs.gentoo.org/905999
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -31,6 +31,7 @@ target_compile_definitions(
+     PUBLIC $<$<CONFIG:Debug>:TINYXML2_DEBUG>
+     INTERFACE $<$<BOOL:${BUILD_SHARED_LIBS}>:TINYXML2_IMPORT>
+     PRIVATE $<$<CXX_COMPILER_ID:MSVC>:_CRT_SECURE_NO_WARNINGS>
++    PUBLIC _FILE_OFFSET_BITS=64
+ )
+ 
+ set_target_properties(
+--- a/Makefile
++++ b/Makefile
+@@ -10,7 +10,7 @@ ARFLAGS = cr
+ RM = rm -f
+ RANLIB = ranlib
+ MKDIR = mkdir -p
+-CXXFLAGS = -fPIC
++CXXFLAGS = -D_FILE_OFFSET_BITS=64 -fPIC
+ 
+ INSTALL = install
+ INSTALL_PROGRAM = $(INSTALL)
+--- a/meson.build
++++ b/meson.build
+@@ -41,6 +41,10 @@ if get_option('debug')
+     tinyxml_extra_args += '-DTINYXML2_DEBUG'
+ endif
+ 
++if host_machine.system() == 'linux'
```

Comment:
Could you update this with the version you sent upstream? (where this bit got dropped as meosn already does it for us)

Reply:
Shoot, I never did that? Sorry! Will update now.